apps/nextjs/src/components/AppComponents/Chat/Chat/hooks/useAilaStreamingStatus.ts

```diff
@@ -16,46 +16,39 @@ export const useAilaStreamingStatus = ({
   isLoading: boolean;
   messages: Message[];
 }): AilaStreamingStatus => {
-  const possibleTypePatch = `"type":"patch"`;
-  const possibleTypePrompt = `"type":"prompt"`;
-
+  const prevStatusRef = useRef<AilaStreamingStatus | null>(null);
   const ailaStreamingStatus = useMemo<AilaStreamingStatus>(() => {
+    const moderationStart = `MODERATION_START`;
+    const chatStart = `CHAT_START`;
+    if (messages.length === 0) return "Idle";
     const lastMessage = messages[messages.length - 1];
-    const contentSplitByRs = lastMessage?.content.split("␞");
-    const lastSectionOfStreamingContent = contentSplitByRs
-      ? contentSplitByRs[contentSplitByRs?.length - 1]
-      : "";
+
     if (isLoading) {
       if (!lastMessage) return "Loading";
+      const { content } = lastMessage;
       if (lastMessage.role === "user") {
         return "RequestMade";
+      } else if (content.includes(moderationStart)) {
+        return "Moderating";
       } else if (
-        lastMessage.role === "assistant" &&
-        lastSectionOfStreamingContent?.includes(possibleTypePatch)
-      ) {
-        return "StreamingLessonPlan";
-      } else if (
-        lastMessage.content.includes(possibleTypePatch) &&
-        lastMessage.content.includes(possibleTypePrompt) &&
-        lastSectionOfStreamingContent !== ""
+        content.includes(`"type":"prompt"`) ||
+        content.includes('"type":"prompt"')
```

Comment:
These two strings look the same to me. Am I missing something?

Reply:
Ah - this was where I was originally testing for escaped strings too \"type\" - looks like that reverted somehow

Reply:
Looks like the linter removed these on save. Resolved.